.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,56 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: true
```

Comment:
I don't think email should be required, some people may not want to share it and we can already see their GitHub username. Thoughts?

Reply:
sure, I buy that